external/netcoreapp/netcoreapp.depproj

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <BinPlaceRef>true</BinPlaceRef>
+    <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
+    <NETCoreAppPackageVersion>2.0.0-preview2-25407-01</NETCoreAppPackageVersion>
```

Comment:
Yeah, we do. I don't know why I only looked on nuget.org for this version.

Will fix.